setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name="pvHelpers",
-    version="5.1.9",
+    version="5.2.0",
```

Comment:
@hu13 would this not be `5.1.10`? Doesn't seem like that big of a change

Reply:
lol, for some reason, my brain is limited to counting up to 10 and does a jump. 🍡

Reply:
haha, makes sense!